### ISSUE TITLE
[CDC] Potential violation in SPI DEVICE

### ISSUE THREAD

**Issue by jeoongp** [lowRISC/opentitan#16643]
There is a direct path from cmd_info_i to cmd_info_o as follows.
cmd_info_i comes from IO_DIV4 and cmd_info_o is on SPI clock domain.

![Screen Shot 2022-12-01 at 4 27 21 PM](https://user-images.githubusercontent.com/103145460/205187673-7b946b5c-e915-44ad-8147-dc43e8dfce18.png)

![Screen Shot 2022-12-01 at 4 18 53 PM](https://user-images.githubusercontent.com/103145460/205187687-9ebbae33-a5dc-4bca-8342-fc45e06303d7.png)

cmd_info_i can be latched to cmd_info_q but I am not sure if one stage latch is OK with CDC.

**Comment by a-will** [comment]
Just an FYI: cmd_info from the CSRs is expected to be quasi-static. Software is supposed to have them set up before SPI clock may begin switching.

There may be some robustness concerns for cases where OT can't make that guarantee (since OT can only set that up with out-of-band control, such as control of the host's reset pin), but I'm not sure if that is a consideration.

**Comment by jeoongp** [comment]
@a-will thanks for info. @eunchan can you review?

**Comment by tjaychen** [comment]
@tjaychen @msfschaffner FYI

**Comment by eunchan** [comment]
yep @a-will is right. similar to other configs, SW configures the command information data prior to enable the SPI_DEVICE.

**Comment by jeoongp** [comment]
@eunchan thanks for confirmation. I added this waiver in https://github.com/lowRISC/opentitan/pull/16673.
